james-agent-io/src/main/java/com/tomtom/james/store/io/ConfigParserWriter.java

```diff
@@ -0,0 +1,17 @@
+package com.tomtom.james.store.io;
+
+import com.tomtom.james.common.api.informationpoint.InformationPoint;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collection;
+
+public interface ConfigParserWriter {
+
+    boolean supportsConfigFile(final String name);
+
+    Collection<InformationPointDTO> parseConfiguration(final InputStream inStream, ScriptsStore fileScriptStore)
+        throws IOException;
+
+    void storeConfiguration(final OutputStream outputStream, Collection<InformationPoint> informationPoints) throws IOException;
```

Comment:
maybe read and write should be separated, I see only one implementation of store

Reply:
Good idea - I'll split this.